DigitalLearningSolutions.Web/Views/Home/Index.cshtml

```diff
@@ -1,16 +1,111 @@
-﻿@{
-  ViewData["Title"] = "Home";
-}
-
-<div class="nhsuk-grid-row">
-  <div class="nhsuk-grid-column-full">
-    <h1 id="page-heading">Home</h1>
-
-    <a class="nhsuk-button" asp-controller="Login" asp-action="Index">
-      Login
-    </a>
-    <a class="nhsuk-button nhsuk-button--secondary" asp-controller="Register" asp-action="Index">
-      Register
-    </a>
-  </div>
-</div>
+﻿@{
+  ViewData["Title"] = "Home";
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/home.css")" asp-append-version="true"/>
+
+<div class="nhsuk-grid-row">
+  <div id="column" class="nhsuk-grid-column-full">
+    <h1 id="page-heading">A platform for digital learning delivery</h1>
+
+    <p id="page-subheading">Enabling health and care organisations to deliver digital learning to staff and students.</p>
+
+    <a class="nhsuk-button auth-button auth-button--left" asp-controller="Login" asp-action="Index">
+      Sign in
+    </a>
+    <a class="nhsuk-button nhsuk-button--secondary auth-button" asp-controller="Register" asp-action="Index">
+      Register
+    </a>

```

Comment:
The scope in the ticket mentions "If the user is logged in, the Login and Registration buttons should be replaced with an “Application selector” button that takes the user to the application selector on the old site." Is that going to be handled in another ticket?